spec/factories/user.rb

```diff
@@ -8,5 +8,13 @@
     trait :confirmed do
       confirmed_at { Faker::Date.backward(4) }
     end
+
+    transient do
```

Comment:
use a trait `:with_targets` for this purpose to avoid create targets every time you call the factory with create